src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,11 +1,29 @@
 package core.basesyntax.service;
 
+import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_LENGTH_PASSWORD = 6;
+    private static final int MIN_AGE_USER = 18;
 
     @Override
     public User register(User user) {
-        return null;
+        StorageDaoImpl storageDao = new StorageDaoImpl();
+        String login = user.getLogin();
+        String password = user.getPassword();
+        int age = user.getAge();
```

Comment:
do we need those variables? we use them only once

Reply:
variables are needed for better readability of the code